doc/pravega-options.md

```diff
@@ -14,3 +14,19 @@ spec:
       metrics.statsdPort: "8125"
 ...
 ```
+### Pravega JVM Options
+
+It is also possible to tune the JVM options for Pravega Controller and Segmentstore. Pravega JVM options are for configuring Controller&Segmenstore JVM process whereas Pravega options are for configuring Pravega software.
+
+Here is an example,
+```
+...
+spec:
+  pravega:
+    controllerJvmOptions: ["-XX:MaxDirectMemorySize=1g"]
+    segmentStoreJVMOptions: ["-XX:MaxDirectMemorySize=1g"]
+...
+```
+There are a bunch of default options in the Pravega operator code that is good for general deployment, please check [here](https://github.com/pravega/pravega-operator/blob/master/pkg/controller/pravega/pravega_controller.go#L175) for controller and [here](https://github.com/pravega/pravega-operator/blob/master/pkg/controller/pravega/pravega_segmentstore.go#L161) for segmentstore. It is possible to override those default values by
```

Comment:
Again no pointers to code, pls. 
Lets mention the default options in the documentation.